bin/s2nc.c

```diff
@@ -115,6 +118,8 @@ static void setup_s2n_config(struct s2n_config *config, const char *cipher_prefs
 
     GUARD_EXIT(s2n_config_set_cipher_preferences(config, cipher_prefs), "Error setting cipher prefs");
 
+    GUARD_EXIT(s2n_config_set_ecc_preferences(config, ecc_prefs), "Error setting ecc prefs");
```

Comment:
if someone uses --tls13 without setting --curves, it would result with bad defaults. either set this when --curves is set or when ecc_prefs != default? same for s2nd

Reply:
Will add the `ecc_prefs != "default"` conditional in s2nd and s2cb.

Reply:
Currently, If `"--tls13"` is enabled without setting curves they are requesting the `"default" `curves list (p-256 and p-384)  with TLS1.3, which is a valid choice but not the default option for TLS1.3. 

I can change this to, If `"--tls13"` is enabled without setting curves they are requesting the `"default_tls13" `curves list (x25519, p-256 and p-384).

Reply:
Connected offline and decided to make a new PR with the changes.